src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,50 @@
+package duke.task;
+
+/**
+ * the task in the list
```

Comment:
Perhaps need a full stop at the end? 🤔
And I noticed the same issue in several other places below.